src/utilities/fe_dofs.jl

```diff
@@ -0,0 +1,66 @@
+type FEField
+    name::AbstractString # Descriptive string
+    dim::Int # Number of dimensions for the field
+end
+
+type FEDofs
```

Comment:
Maybe the FEDofs should have a `FEMesh` field. I am thinking that `FEDofs` is pretty useless without it's corresponding mesh so if you want to use `FEDofs` you will also always have to carry around `FEMesh`? If that is the case then maybe just putting the mesh inside the dof object makes sense.